ci/test_wheel_cugraph-pyg.sh

```diff
@@ -33,11 +33,13 @@ if [[ "${CUDA_VERSION}" == "11.8.0" ]]; then
     -f https://data.pyg.org/whl/torch-2.1.0+cu118.html
 
   rapids-logger "pytest cugraph-pyg (single GPU)"
+  pushd python/cugraph-pyg/cugraph_pyg
   python -m pytest \
     --cache-clear \
     --ignore=tests/int \
```

Comment:
I copied this code block over from `test_python.sh`. It seems that we no longer have `tests/int` directory. Can I safely remove this line from both places? @alexbarghi-nv

Reply:
yes